cmake/tribits/ctest_driver/TribitsCTestDriverCore.cmake

```diff
@@ -1004,31 +1004,46 @@ INCLUDE(${CMAKE_CURRENT_LIST_DIR}/TribitsCTestDriverCoreHelpers.cmake)
 # and other results and submitted and displayed on CDash (but not what CDash
 # site(s) or project(s) they are submitted to).  These options can all be set
 # in the CTest -S script using ``SET()`` statements before
-# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env.
+# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env when
+# running the CTest -S driver script.
 #
 #   .. _CTEST_TEST_TYPE:
 #
 #   ``CTEST_TEST_TYPE=[Nightly|Continuous|Experimental]``
 #
-#     Determines the type of build.  This value is passed in as the first
+#     Determines the model for build.  This value is passed in as the first
 #     argument to the built-in CTest function ``CTEST_START()``.  Valid values
-#     include ``Nightly``, ``Continuous``, and ``Experimental``.  This also
-#     defines the default value for `${PROJECT_NAME}_TESTING_TRACK`_ as well
-#     as defines the default value for
+#     include ``Nightly``, ``Continuous``, and ``Experimental``.  As far a
+#     CTest is concerned, the only real impact this CTest "Model" has is on
+#     setting the time stamp in the build stamp (which is stored in the file
+#     ``Testing/TAG``).  For the model ``Nightly``, the time stamp in the
+#     build stamp is taken from the variable ``CTEST_NIGHTLY_START_TIME`` read
+#     in from the file `<projectDir>/CTestConfig.cmake`_ file.  Otherwise, the
+#     time stamp used is the current build start time.  (The reason this is
+#     significant is that builds on CDash that have the same site, buildname,
+#     and buildstamp are considered the same build and will combine results.)
+#     This also defines the default value for `${PROJECT_NAME}_TRACK`_ (see
+#     below) as well as defines the default value for
```

Comment:
Sometimes the `*stamp` words are two words (`build stamp`) and sometimes they're combined (`buildstamp`).

Reply:
Addressed in commit https://github.com/bartlettroscoe/TriBITS/commit/c38b89ca8c836b4f9d9ded010b9825aa959b39e0